src/index.js

```diff
@@ -5,18 +5,18 @@
 
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { BrowserRouter } from 'react-router-dom';
+import { BrowserRouter, HashRouter } from 'react-router-dom';
```

Comment:
can remove BrowserRouter if it's not being used anymore

Reply:
good point, will remove it